client/components/post-card/search.jsx

```diff
@@ -53,7 +53,7 @@ export function SearchPostCard( { post, site, feed, onClick = noop, onCommentCli
 					commentCount={ post.discussion.comment_count }
 					tagName="span" showLabel={ false }
 					onClick={ onCommentClick }/>
-				<LikeButton siteId={ post.site_ID } postId={ post.ID } tagName="span" showCount={ true } showLabel={ false } />
+				<LikeButton siteId={ post.site_ID } postId={ post.ID } tagName="span" showCount={ false } showLabel={ false } />
```

Comment:
No worries. Captured in https://github.com/Automattic/wp-calypso/issues/6767 so we don't forget.